patches/buildroot.diff

```diff
@@ -85,24 +87,47 @@ index 7375c29a74..ff057b2955 100644
  BR2_PACKAGE_WPA_SUPPLICANT=y
  BR2_PACKAGE_WPA_SUPPLICANT_CLI=y
  BR2_PACKAGE_WPA_SUPPLICANT_PASSPHRASE=y
-+BR2_PACKAGE_OPKG=y
-+BR2_PACKAGE_NANO=y
++BR2_PACKAGE_OPKG=n
++BR2_PACKAGE_NANO=n
  BR2_PACKAGE_POLL_SYSFS=y
  BR2_PACKAGE_AD936X_REF_CAL=y
  BR2_TARGET_ROOTFS_CPIO=y
-@@ -63,3 +79,15 @@ BR2_TARGET_ROOTFS_CPIO_UIMAGE=y
+@@ -62,3 +78,15 @@ BR2_TARGET_ROOTFS_CPIO_UIMAGE=y
  BR2_PACKAGE_HOST_DOSFSTOOLS=y
  BR2_PACKAGE_HOST_GENIMAGE=y
  BR2_PACKAGE_HOST_MTOOLS=y
-+BR2_PACKAGE_FFMPEG_43=y
++BR2_PACKAGE_FFMPEG_43=n
```

Comment:
I had to disable these multi-media packages, otherwise the image gets too big, even without the licenses.